src/app/rpmostree-builtin-status.cxx

```diff
@@ -1248,15 +1252,22 @@ print_history_entry (RORHistoryEntry  *entry,
         }
 
       json_builder_set_member_name (builder, "deployment-create-timestamp");
-      json_builder_add_int_value (builder, entry->deploy_timestamp);
+      json_builder_add_int_value (builder, entry.deploy_timestamp);
       json_builder_set_member_name (builder, "deployment-create-command-line");
-      json_builder_add_string_value (builder, entry->deploy_cmdline);
+      if (entry.deploy_cmdline.length() > 0)
+        {
+          // Copy so we can use c_str()
+          auto cmdline_copy = std::string(entry.deploy_cmdline);
+          json_builder_add_string_value (builder, cmdline_copy.c_str());
+        }
+      else
+        json_builder_add_string_value (builder, "");
```

Comment:
This is passing an empty string, where previously we passed `NULL` if no cmdline was found.

Hmm, I think what we had here was wrong anyway. I think we want something like this:

```
if (entry.deploy_cmdline.length() > 0)
  {
    json_builder_set_member_name (builder, "deployment-create-command-line"); 
    // Copy so we can use c_str()
    auto cmdline_copy = std::string(entry.deploy_cmdline);
    json_builder_add_string_value (builder, cmdline_copy.c_str()); 
  }
```

? (IOW we don't even emit that field if there's no value.)